src/app.js

```diff
@@ -1 +1,45 @@
 'use strict';
+
+const { numberGenerator } = require('./numberGenerator');
+const { terminal } = require('./terminal');
+const { calculator } = require('./calculator');
+const { output } = require('./output');
+
+const { INPUT_MESSAGE, OUTPUT_MESSAGE } = require('./constants');
+
+const generatedNums = numberGenerator();
+
+const getNumber = (userInput) => {
+  if (userInput.length !== 4) {
+    terminal.question(
+      INPUT_MESSAGE.enterCorrectNumber,
+      num => getNumber(num)
+    );
+  } else if ((/0-9/).test(userInput)) {
+    terminal.question(
+      INPUT_MESSAGE.enterCorrectNumber,
+      num => getNumber(num)
+    );
+  } else {
+    const message = output(calculator(generatedNums, userInput));
+
+    if (message === OUTPUT_MESSAGE.win) {
+      // eslint-disable-next-line no-console
```

Comment:
If you know that you will need to use many console logs, you can disable it properly for the linter once for the entire file